### PR TITLE
[mimir-rules-action] fix multiline output expression

### PR DESCRIPTION
**Pull request by rasta-rocket**
Signed-off-by: rasta-rocket <bruno.fernando@jobteaser.com>

#### What this PR does

Hey folks 👋

Small PR that fix/simplify the way the multiline is rendered in the output of the action mimir-rules-action

With the following pipeline:
```
name: mimir_rules_ci
on:
  workflow_dispatch:
  pull_request:
jobs:
  mimir_rules:
    name: Check Mimir Rules
    runs-on: ubuntu-latest
    steps:
      - name: Checkout
        uses: actions/checkout@v4
      - name: Check Mimir rules
        uses: rasta-rocket/mimir/operations/mimir-rules-action@action_mimirtool_fix_multiline
        env:
          MIMIR_ADDRESS: "XXXXX"
          MIMIR_TENANT_ID: XXXXX
          ACTION: check
          RULES_DIR: "./mimir/alert,./mimir/record"
      - name: Diff Mimir rules
        id: diff_rules
        uses: rasta-rocket/mimir/operations/mimir-rules-action@action_mimirtool_fix_multiline
        env:
          MIMIR_ADDRESS: "XXXXX"
          MIMIR_TENANT_ID: XXXXX
          ACTION: diff
          RULES_DIR: "./mimir/alert,./mimir/record"
      - name: Put diff in summary
        run: |
           echo "${{ steps.diff_rules.outputs.detailed }}"

           echo '# Diff summary' >> $GITHUB_STEP_SUMMARY
           echo '```diff' >> $GITHUB_STEP_SUMMARY
           echo "${{ steps.diff_rules.outputs.detailed }}" >> $GITHUB_STEP_SUMMARY
           echo '```' >> $GITHUB_STEP_SUMMARY
```

This has been done following : https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions#example-of-a-multiline-string

Don't hesitate to ping me if any remarks/questions 😉 ?

Cheers ☀️
#### Which issue(s) this PR fixes or relates to

Fixes #<issue number>

#### Checklist

- [ ] Tests updated.
- [ ] Documentation added.
- [ ] `CHANGELOG.md` updated - the order of entries should be `[CHANGE]`, `[FEATURE]`, `[ENHANCEMENT]`, `[BUGFIX]`.
- [ ] [`about-versioning.md`](https://github.com/grafana/mimir/blob/main/docs/sources/mimir/configure/about-versioning.md) updated with experimental features.

### CODE REVIEW

**Comment by rasta-rocket** [comment]
@56quarters 👋 

I'm trying my luck with you to get a review on this 😅

File: operations/mimir-rules-action/entrypoint.sh [hunk 1]
```diff
@@ -106,8 +106,7 @@ case "${ACTION}" in
     ;;
 esac
 
-SINGLE_LINE_OUTPUT=$(echo "${OUTPUT}" | awk 'BEGIN { RS="%0A" } { gsub(/%/, "%25"); gsub(/\r/, "%0D"); gsub(/\n/, "%0A") } { print }')
-echo "detailed=${SINGLE_LINE_OUTPUT}" >> "${GITHUB_OUTPUT}"
+{ echo "detailed<<EOF"; echo "$OUTPUT"; echo "EOF"; } >> "$GITHUB_OUTPUT"
```
Comment (colega) [inline operations/mimir-rules-action/entrypoint.sh:109]: Is there an equal missing?
```suggestion
{ echo "detailed=<<EOF"; echo "$OUTPUT"; echo "EOF"; } >> "$GITHUB_OUTPUT"
```

Comment (colega) [reply operations/mimir-rules-action/entrypoint.sh:109]: I'm confused because the summary below is also missing the equal 🤔

Comment (colega) [reply operations/mimir-rules-action/entrypoint.sh:109]: Okay, I've read the docs and this seems to be correct 👍

**Review by colega — APPROVED** [review]
Thank you! Sorry for such a long delay for the review.

**Comment by colega** [comment]
I'm trying to merge this but for some reason the CI won't run, can I ask you to rebase this on latest main and force-push?

**Comment by rasta-rocket** [comment]
@colega sorry for the late answer and thanks a lot for the review 🚀 

Any chance you can do a review on a similar PR: https://github.com/grafana/cortex-rules-action/pull/27 ?

**Comment by colega** [comment]
Reviewed, thanks!

**Pull request merged**